### PR TITLE
Make api-generator scripts generic

### PR DESCRIPTION
**Pull request by GokceGK**

### CODE REVIEW

File: scripts/generate-sdk/generate-sdk.sh [hunk 1]
```diff
@@ -16,6 +22,31 @@ SCRIPTS_FOLDER="${SDK_REPO_LOCAL_PATH}/scripts"
 GENERATOR_VERSION="v6.6.0"
 GENERATOR_VERSION_NUMBER="${GENERATOR_VERSION:1}"
 
+if [[ -z ${GIT_HOST} ]]; then
+    echo "Git host is empty, default will be used."
+    GIT_HOST=${GIT_HOST_DEFAULT}
+fi
```
Comment (joaopalet) [inline scripts/generate-sdk/generate-sdk.sh:28]: This one also needs to be used in the call to the generator

File: scripts/sdk-create-pr.sh [hunk 2]
```diff
@@ -6,11 +6,10 @@ ROOT_DIR=$(git rev-parse --show-toplevel)
 COMMIT_NAME="SDK Generator Bot"
 COMMIT_EMAIL="noreply@stackit.de"
 SDK_REPO_LOCAL_PATH="${ROOT_DIR}/sdk-repo-updated" # Comes from generate-sdk.sh
-REPO_URL_SSH="git@github.com:stackitcloud/stackit-sdk-go.git"
 REPO_BRANCH="main"
 
-if [ $# -ne 4 ]; then
-    echo "Not enough arguments supplied. Required: 'branch-name' 'commit-message' 'pr-title' 'pr-body'"
+if [ $# -ne 5 ]; then
+    echo "Not enough arguments supplied. Required: 'branch-name' 'commit-message' 'pr-title' 'pr-body' 'repo-url'"
```
Comment (joaopalet) [inline scripts/sdk-create-pr.sh:12]: This one doesn't need to be required, right?

Comment (joaopalet) [reply scripts/sdk-create-pr.sh:12]: The fallback is already below

File: scripts/generate-sdk/generate-sdk.sh (hunk 1 above)
Comment (GokceGK) [reply scripts/generate-sdk/generate-sdk.sh]: Added it to the generator call

File: scripts/sdk-create-pr.sh (hunk 2 above)
Comment (GokceGK) [reply scripts/sdk-create-pr.sh:12]: Changed condition to lt 4 as it should be at least 4 but can also be 5 :)

**Review by joaopalet — APPROVED** [review]

**Pull request merged**